### ISSUE TITLE
InvoicePlane Android app

### ISSUE THREAD

**Issue by Tiam-Abderezai** [InvoicePlane/InvoicePlane#819]
I noticed there isn't a repo for an Android version of InvoicePlane. Can I make one? Would like to contribute.

**Comment by clockwiseq** [comment]
You mean like a native app for Android?  InvoicePlane is a web app only currently.  If you want to contribute and create an Android app, that would be fantastic!

**Comment by nielsdrost7** [comment]
@Tiam-Abderezai I've created a repository and added you to it.

**Comment by Tiam-Abderezai** [comment]
@nielsdrost7 @clockwiseq Thanks!!! I'm going to start working on it today. I need to do a couple of things before I fully embark on this expedition:

- [ ] Install and configure the InvoicePlane server locally
- [ ] Checkout the Web UI/UX and get an understanding of everything
- [ ] Read through InvoicePlane contribution guide to make sure I follow them thoroughly
- [ ] Create ReadMe file
- [ ] Create wireframe/story boards for the first prototype of the Android UI and share it with you and others to discuss and get your feedback
- [ ] Submit the first PR for the first native Android app for InvoicePlane

**Comment by nielsdrost7** [comment]
@Tiam-Abderezai Come find us on Slack before you start. I think I can save you some time.
https://join.slack.com/t/invoiceplane/shared_invite/zt-sjirc9h2-952baR9rn4r8LyzRK9MveA

**Comment by Tiam-Abderezai** [comment]
@JohnDS1 Absolutely! I'm just setting up the testing environment right now to lay the groundworks to ensure a solid foundation for the App. I'll setup the first master branch for the Android app tonight @nielsdrost7 If it's ok with you and have your permission.

@JohnDS1 what are some areas in Android (Kotlin) you'd like to work on? I'm compiling a list of tech-stack, patterns, practices, dependencies, etc

**Comment by nielsdrost7** [comment]
@JohnDS1 I've added you to the guthub repo.
When I have time I'll make a separate slack channel for the Android app.

**Comment by JohnDS1** [comment]
@Tiam-Abderezai I currently have ~4 years of android experience and a few months of Kotlin experience. I don't mind working on anything.

@nielsdrost7 Thanks for adding me to the repo. I look forward to the slcak channel.
